src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -898,6 +899,27 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                             printFullPaths = true;
                             continue;
 
+                        case "pathmap":
+                            // "/pathmap:K1=V1,K2=V2..."
```

Comment:
I think it is appropriate to do this right from the start. I think it is mandatory for this change.